cal/cal/static/templates/calendar-filter.html

```diff
@@ -0,0 +1,26 @@
+<div ng-controller="CalendarCtrl as $ctrl" class="cal-calendar col-md-8">
```

Comment:
what does this file do? and since tagdetails also has mostly the same thing could you make a base html file that they inherit from?